hack/make-rules/test-cmd-util.sh

```diff
@@ -1356,6 +1356,17 @@ run_kubectl_get_tests() {
   kube::test::if_has_string "${output_message}" "/apis/extensions/v1beta1/namespaces/default/deployments 200 OK"
   kube::test::if_has_string "${output_message}" "/apis/extensions/v1beta1/namespaces/default/replicasets 200 OK"
 
+  ### Test kubectl get chunk size
+  output_message=$(kubectl --v=6 get clusterrole --chunk-size=10 2>&1 "${kube_flags[@]}")
+  # Post-condition: Check if we get a limit and continue
+  kube::test::if_has_string "${output_message}" "/apis/rbac.authorization.k8s.io/v1/clusterroles?limit=10 200 OK"
+  kube::test::if_has_string "${output_message}" "/apis/rbac.authorization.k8s.io/v1/clusterroles?continue="
```

Comment:
omit `/apis/rbac.authorization.k8s.io/v1`... we don't care what version, just the query params